vars/buildXamarinIos.groovy

```diff
@@ -0,0 +1,35 @@
+def call(Map config) {
+
+    String solution = config.solution ?: findSolution()
```

Comment:
Can we log a message here indicating which solution file is being used - just thinking if we call `findSolution()` and we have git submodules which contain a solution file and we incorrectly detect that file, it would make debugging easier.

Reply:
yep great shout. added